flink-runtime/src/test/java/org/apache/flink/runtime/minicluster/TestingMiniCluster.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.minicluster;
+
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.runtime.highavailability.HighAvailabilityServices;
+
+import javax.annotation.Nonnull;
+
+import java.util.concurrent.Executor;
+import java.util.function.Supplier;
+
+/**
+ * {@link MiniCluster} extension which allows to set a custom {@link HighAvailabilityServices}.
+ */
+public class TestingMiniCluster extends MiniCluster {
+
+	@Nonnull
+	private final Supplier<HighAvailabilityServices> highAvailabilityServicesSupplier;
+
+	public TestingMiniCluster(
+		MiniClusterConfiguration miniClusterConfiguration,
+		@Nonnull Supplier<HighAvailabilityServices>highAvailabilityServicesSupplier) {
```

Comment:
space between  `Supplier<HighAvailabilityServices>` and `highAvailabilityServicesSupplier`
and maybe newline thereafter, before the first line of function code

Reply:
will change it.